src/lib/process/api.ts

```diff
@@ -84,18 +84,24 @@ export const disableProcessTransaction = async (
 type GetAllFn = (options: Polkadot.Options) => Promise<Process.Payload[]>
 
 export const getAll: GetAllFn = async (options) => {
-  const polkadot: Polkadot.Polkadot = await api.createNodeApi(options)    
+  const polkadot: Polkadot.Polkadot = await api.createNodeApi(options)
   const processes = await Promise.all(
-    (await polkadot.api.query.processValidation.processModel.entries())
-      .map(([id, data]: any) => new Promise((r) => {
-        getVersion(polkadot, id).then((version) => {
-          return r({
-            id,
-            version,
-            ...data
+    (
+      await polkadot.api.query.processValidation.processModel.entries()
+    ).map(
+      ([id, data]: any) =>
+        new Promise((r) => {
+          getVersion(polkadot, id).then((version) => {
+            return r({
+              id,
+              version,
+              status: data.status,
+              program: JSON.stringify(data.program),
```

Comment:
PR updates a major new version of `@polkadot/api`and the format returned by `polkadot.api.query.processValidation.processModel.entries()` is different. No more `Getter` and `Setter`

before
```
Type(2) [Map] {
  'status' => Type {
    registry: TypeRegistry {},
    isDisabled: [Getter],
    isEnabled: [Getter],
    asDisabled: [Getter],
    asEnabled: [Getter]
  },
  'program' => Type(1) [
    Type {
      registry: TypeRegistry {},
      isOp: [Getter],
      isRestriction: [Getter],
      asOp: [Getter],
      asRestriction: [Getter]
    },
    registry: TypeRegistry {},
    initialU8aLength: 3
  ],
  status: [Getter],
  program: [Getter],
}
 ```
 
 after
 ```
Type(2) [Map] {
  'status' => Type { registry: TypeRegistry {} },
  'program' => Type(1) [
    Type { registry: TypeRegistry {} },
    registry: TypeRegistry {},
    initialU8aLength: 3
  ],
}
 ```